src/controller/CMakeLists.txt

```diff
@@ -17,6 +17,8 @@ set(SOURCES
         )
 
 add_library(${PROJECT_NAME} ${SOURCES})
+
+find_package(Boost COMPONENTS system REQUIRED)
```

Comment:
Niet nodig als je dit in de root CmakeLists doet

Reply:
Dan kunnen we alles daar wel in zetten :p